app/assets/src/components/views/PipelineViz.jsx

```diff
@@ -0,0 +1,379 @@
+import React from "react";
+import { forEach } from "lodash";
```

Comment:
Please use `lodash/fp`. It is our team convention to avoid using the same library with two different paradigms. 
It follows a functional programming paradigm, were the subject is typically the last argument.